bigquery/google/cloud/bigquery/job.py

```diff
@@ -1949,6 +1949,15 @@ def result(self, timeout=None, retry=DEFAULT_RETRY):
         return self._client.list_rows(dest_table, selected_fields=schema,
                                       retry=retry)
 
+    def to_dataframe(self):
+        import pandas as pd
+
+        iterator = self.result()
```

Comment:
`results`? (`iterator` is a really generic term, better to name it after *what* it iterates over).

Reply:
Changed to `query_results` in 4bc85fd